webapp/views.py

```diff
@@ -382,7 +382,7 @@ def get_notice(notice_id, **kwargs):
     if not kwargs.get("show_hidden", False):
         notice_query = notice_query.filter(Notice.is_hidden == "False")
 
-    notice = notice_query.filter(Notice.id == notice_id).one_or_none()
+    notice = notice_query.filter(Notice.id == notice_id.upper()).one_or_none()
 
```

Comment:
Why not use `get(I'd)` here, like with CVEs?

Reply:
It throws this error when I try to do `get(id)`:  `Query.get() being called on a Query with existing criterion.` From what I've read it's because the query has already been filtered on line 383...but I don't entirely understand why    `notice = notice_query.filter(Notice.id == notice_id.upper()).one_or_none()` doesn't throw the same error for the same reason